vendor/bat-native-ledger/src/bat/ledger/internal/wallet/create.h

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_WALLET_CREATE_H_
+#define BRAVELEDGER_WALLET_CREATE_H_
+
+#include <stdint.h>
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "bat/ledger/ledger.h"
+#include "bat/ledger/internal/bat_helper.h"
+
+namespace bat_ledger {
+class LedgerImpl;
+}
+
+namespace braveledger_wallet {
+
+class Create {
```

Comment:
Really minor nit: Some of these class names seem a bit vague (`Create` doesn't tell you too much if you see it in isolation; similar with `Recover`). I guess you're trying to avoid the duplication of using `Wallet` since that's already in the namespace. But, I think I would call these something like `WalletCreation` and `WalletRecovery`, if possible.